.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,8 @@
     depending on your change. Keep or delete what is relevant for your change.
     Remember that it helps us review if you give more helpful info for us to
     understand your change.
+    
+    Ensure that you've read through and followed the [Contributing Guidelines](CONTRIBUTING.md).
```

Comment:
As a note, since this is a plaintext document, hyperlinks don't work in the issue / PR templates. I would suggest putting a link to the contributing guidelines on a new line right below.